lib/FullScreenView/FullScreenView.js

```diff
@@ -32,24 +29,32 @@ export const FullScreenView = ({
             data-test-header-close-button
             ariaLabel={ariaLabel}
             icon="times"
-            ref={headerCloseButtonRef}
             onClick={onCancel}
           />
         )}
       </FormattedMessage>
     </PaneMenu>
   );
 
+  const renderPaneHeader = renderProps => (
+    <PaneHeader
+      {...renderProps}
+      paneTitle={paneTitle}
+      actionMenu={actionMenu}
+      firstMenu={firstMenu}
+    />
+  );
+
   return (
     <div
       id={id}
       data-test-full-screen-view
+      {...props}
     >
       <Pane
+        id={paneId}
```

Comment:
As far as I understand this property is used only for testing purposes. Are there any specific cases when the line 50
( id={id}) doesn't cover needed behavior?

Reply:
I added this props as it was passed to the Pane component from data-import components

Reply:
Yes, that is used in data-import to identify the pane. My question is if we need extra property here or can use the current approach. As I mentioned before, it seems it is used only for testing purposes to identify the pane scope. Is ui-data-import broken if you use div id instead of pane id?

Reply:
There is different ids pass for different profiles.

Reply:
Yes, you are right. And you will still have a chance to add a custom id like I mentioned before.

Reply:
If this id is for testing purposes I think it can be replaced with data-test attribute in data-import project and thus we have fewer props in this component to manage.

Reply:
ok, agree with you